pages/api/performance/get.js

```diff
@@ -0,0 +1,28 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get performances by specifying the corresponding eventID
+// POST with { eventID: id }
```

Comment:
This word "POST" in this comment is a little confusing (similar in the files above as well).